client/app/dbaas/logs/translations/Messages_de_DE.json

```diff
@@ -801,4 +797,4 @@
  "logs_member_delete_question": "Sind Sie sicher, dass Sie das Mitglied <strong>{{username}}</strong> entfernen möchten?",
  "logs_role_member_remove_success": "Das Mitglied {{name}} wurde erfolgreich entfernt.",
  "logs_role_member_remove_error": "Beim Entfernen des Mitglieds ist ein Fehler aufgetreten: {{message}}"
-}
\ No newline at end of file
+}
```

Comment:
We should avoid updating other translations files than `fr_FR`

Reply:
@marie-j You're right! (See: https://github.com/ovh-ux/ovh-manager-cloud/blob/master/CONTRIBUTING.md#translations).

Reply:
All non-FR files are reverted